blackjack.rb

```diff
@@ -93,6 +105,13 @@ def player_hit_or_stay(current_hand)
     end
   end
 
+#cpu doesn't handle aces (unless I missed it).
+# I did something like this, but it assumes ace will auto change so you don't bust
+# if hand.any?{ |c| c.face == 'A' }
+#    hand.inject(:+) < 18
```

Comment:
I'm sorry, I don't think that I understand what this is supposed to do. Comparison returns a boolean, so I don't understand what is supposed to happen?